### PR TITLE
docs(observability): quickstart

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: observability/cockpit/quickstart.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+</Message>
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
+
+3. Click the **Settings** tab.
+
+4. Click the **Deactivate Cockpit** button.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:74]:
```suggestion
4. Click **Deactivate Cockpit**.
```

File: observability/cockpit/quickstart.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+7. Click the folder icon. The Scaleway folder displays.
+
+8. Click the Scaleway folder to display Scaleway's managed dashboards.
+
+## How to deactivate your Cockpit
+
+<Message type="important">
+
+ By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted. You can reactivate your Cockpit at any time.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:64]:
```suggestion
 By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.
```

File: observability/cockpit/quickstart.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  h1: Observability - Quickstart
+  paragraph: This page shows you how to get started with Scaleway Observability.
+dates:
+  validation: 2022-10-15
+  posted: 2022-10-04
+---
+
+Scaleway Observability allows you to monitor your applications and their infrastructure by giving you insights and context into their behavior. Observability also enables you to visualize your metrics and logs through a Grafana dashboard.
+
+In this Quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to connect to Grafana and access your [managed dashboards](observability/cockpit/concepts/#managed-dashboards). When you are done using the Cockpit, you can also deactivate it.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:15]:
```suggestion
In this Quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to connect to Grafana and access your [managed dashboards](/observability/cockpit/concepts/#managed-dashboards). When you are done using the Cockpit, you can also deactivate it.
```

File: observability/cockpit/quickstart.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+meta:
+  title: Observability - Quickstart
+  description: This page shows you how to get started with Scaleway Observability.
+content:
+  h1: Observability - Quickstart
+  paragraph: This page shows you how to get started with Scaleway Observability.
+dates:
+  validation: 2022-10-15
+  posted: 2022-10-04
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:10]:
```suggestion
  posted: 2022-10-30
```

File: observability/cockpit/quickstart.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+<Message type="requirement">
+
+ You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+
+</Message>
+
+## How to activate your Cockpit
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:25]: Unnecessary step

Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:25]:
```suggestion
```

File: observability/cockpit/quickstart.mdx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Click **Cockpit** in the side menu.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:27]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+3. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:29]:
```suggestion
2. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.
```

File: observability/cockpit/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+
+## How to retrieve your Grafana credentials
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:34]:
```suggestion
```

File: observability/cockpit/quickstart.mdx [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:36]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+3. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:38]:
```suggestion
2. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```

File: observability/cockpit/quickstart.mdx [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+## How to connect to Grafana and access your managed dashboards
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:42]:
```suggestion
```

File: observability/cockpit/quickstart.mdx [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:44]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+3. Click **Open your dashboard** to open your Grafana managed dashboard. You are redirected to the Grafana website.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:46]:
```suggestion
2. Click **Open your dashboard** to open your Grafana managed dashboard. You are redirected to the Grafana website.
```

File: observability/cockpit/quickstart.mdx [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+4. Log in to Grafana using your [Grafana credentials](observability/cockpit/how-to/retrieve-grafana-credentials/).
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:48]:
```suggestion
3. Log in to Grafana using your [Grafana credentials](observability/cockpit/how-to/retrieve-grafana-credentials/).
```

File: observability/cockpit/quickstart.mdx [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+
+5. Click **Log in**. The Grafana overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:50]:
```suggestion
4. Click **Log in**. The Grafana overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+
+6. Hover over **Dashboards** and click **Browse**.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:52]:
```suggestion
5. Hover over **Dashboards** and click **Browse**.
```

File: observability/cockpit/quickstart.mdx [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+
+    <Lightbox src="scaleway_garafna_browse.webp" alt="" />
+
+7. Click the folder icon. The Scaleway folder displays.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:56]:
```suggestion
6. Click the folder icon. The Scaleway folder displays.
```

File: observability/cockpit/quickstart.mdx [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+
+8. Click the Scaleway folder to display Scaleway's managed dashboards.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:58]:
```suggestion
7. Click the Scaleway folder to display Scaleway's managed dashboards.
```

File: observability/cockpit/quickstart.mdx [hunk 19]
```diff
@@ -0,0 +1,83 @@
...
+
+## How to deactivate your Cockpit
+
+<Message type="important">
+
+ By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.
+
+</Message>
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:68]:
```suggestion
```

File: observability/cockpit/quickstart.mdx [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:70]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 21]
```diff
@@ -0,0 +1,83 @@
...
+
+3. Click the **Settings** tab.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:72]:
```suggestion
2. Click the **Settings** tab.
```

File: observability/cockpit/quickstart.mdx [hunk 22]
```diff
@@ -0,0 +1,83 @@
...
+
+4. Click **Deactivate Cockpit**.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:74]:
```suggestion
3. Click **Deactivate Cockpit**.
```

File: observability/cockpit/quickstart.mdx [hunk 23]
```diff
@@ -0,0 +1,83 @@
...
+
+5. Type **DEACTIVATE** and then click **Deactivate Cockpit** to confirm. Your Cockpit will be deleted in a matter of minutes.
```
Comment (nedaehm) [inline observability/cockpit/quickstart.mdx:76]:
```suggestion
4. Type **DEACTIVATE** and then click **Deactivate Cockpit** to confirm. Your Cockpit will be deleted in a matter of minutes.
```

File: observability/cockpit/quickstart.mdx [hunk 24]
```diff
@@ -0,0 +1,79 @@
...
+<Message type="requirement">
+
+ You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+
+</Message>
+
+## How to activate your Cockpit
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:26]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 25]
```diff
@@ -0,0 +1,79 @@
...
+
+2. Click the **Grafana Users** tab. Your Grafana credentials display. You can copy them and [log in to your Grafana account](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
+
+## How to connect to Grafana and access your managed dashboards
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:41]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/quickstart.mdx [hunk 26]
```diff
@@ -0,0 +1,79 @@
...
+## How to deactivate your Cockpit
+
+<Message type="important">
+
+ By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.
+
+</Message>
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:66]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

**Review by RoRoJ — APPROVED** [review]

File: observability/cockpit/quickstart.mdx [hunk 27]
```diff
@@ -0,0 +1,79 @@
...
+  h1: Observability - Quickstart
+  paragraph: This page shows you how to get started with Scaleway Observability.
+dates:
+  validation: 2022-10-15
+  posted: 2022-10-30
+---
+
+Scaleway Observability allows you to monitor your applications and their infrastructure by giving you insights and context into their behavior. Observability also enables you to visualize your metrics and logs through a Grafana dashboard.
+
+In this Quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to connect to Grafana and access your [managed dashboards](/observability/cockpit/concepts/#managed-dashboards). When you are done using the Cockpit, you can also deactivate it.
```
Comment (ldecarvalho-doc) [inline observability/cockpit/quickstart.mdx:15]: maybe use bullet points?

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**